.NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs

```diff
@@ -22,12 +22,13 @@ public static class BasePhoneNumbers
 		public const string BRPhoneNumberRegex = @"(((\B\(\s?))\d{2,3}(\s?\))|(\b\d{2,3}))\s?\d{4,5}-?\d{3,5}(?!-)\b";
 		public const string UKPhoneNumberRegex = @"(((\b(00)|\B\+)\s?)?(\b\d{2}\s?)?((\s?\(0\)[-\s]?|\b|(?<=(\b^#)\d{2}))\d{2,5}|\(0\d{3,4}\))[/-]?\s?(\d{5,8}|\d{3,4}[-\s]?\d{3,4})(?!-)\b)";
 		public const string DEPhoneNumberRegex = @"((\+\d{2}\s?((\(0\))?\d\s?)?|\b)(\d{2,4}\s?[-/]?[\s\d]{7,10}\d)(?!-)\b)";
-		public const string USPhoneNumberRegex = @"((((\B\+)|\b)1(\s|-)?)|\b)?(\d{3}\)[-\s]?|\(\d{3}\)[-\s]?|\b\d{3}\s?[-\.]?\s?)\d{3}\s?[-\.]?\s?\d{4}(\s?(x|X|ext)\s?\d{3,5})?(?!(-\s?\d))\b";
+		public const string USPhoneNumberRegex = @"(((((\B\+)|\b)1(\s|-)?)|\b)?(\d{3}\)[-\s]?|\(\d{3}\)[-\s]?|\b\d{3}\s?[-\.]?\s?)|\b)\d{3}\s?[-\.]?\s?\d{4}(\s?(x|X|ext)\s?\d{3,5})?(?!(-\s?\d))\b";
 		public const string CNPhoneNumberRegex = @"((\b00\s?)?\+?86\s?-?\s?)?(((\b|(?<=86))\d{2,5}\s?-?\s?|\(\d{2,5}\)\s?)\d{4}\s?-?\s?\d{4}(\s?-?\s?\d{4})?|(\b|(?<=86))\d{3}\s?-?\s?\d{4}\s?-?\s?\d{4})(?!-)\b";
 		public const string DKPhoneNumberRegex = @"((\(\s?(\+\s?|00)45\s?\)\s?)|(((?<!\d)\+\s?|\b00)45\s?)|\b)(\s?\(0\)\s?)?((\d{8})|(\d{4}\s?-?\s?\d{4,6})|((\d{2}[\s-]){3}\d{2})|(\d{2}\s?-?\s?\d{3}\s?-?\s?\d{3}))(?!-)\b";
 		public const string ITPhoneNumberRegex = @"((\(\s?(\+\s?|00)39\s?\)\s?)|(((?<!\d)\+\s?|\b00)39\s?)|\b)((0[\d\s-]{4,12}\d)|(3[\d\s-]{7,12}\d))(?!-)\b";
 		public const string NLPhoneNumberRegex = @"((((\(\s?(\+\s?|00)31\s?\)\s?)|(((?<!\d)\+\s?|\b00)31\s?))?(((\b|(?<=31))0?\d{1,3}|\(\s?0?\d{1,3}\s?\)|\(0\)[-\s]?\d{1,3})\s?-?[\d\s]{5,11}\d))|\b\d{10,12})(?!-)\b";
 		public const string SpecialPhoneNumberRegex = @"\b(\d{3,4}[/-]\d{1,4}[/-]\d{3,4})\b";
+		public const string SpecialUSPhoneNumberRegex = @"(?<!(-|-\s|\d|\)|\)\s|\.))\d{3}\s?[-\.]\s?\d{4}(?!(-\s?\d))\b";
```

Comment:
It's best to rename these new regexes and vars to something like NoAreaCodeUSNumber. "Special" is a very ambiguous term.

Reply:
Also, this regex is for scoring only (not tagging), right? If so, can you add a comment explaining that in the YAML file?

Reply:
Yes, it's only for scoring. I'll add a comment in the YAML file.